cmd/wallet-web/src/pages/VaultsPage.vue

```diff
@@ -3,6 +3,140 @@
  *
  * SPDX-License-Identifier: Apache-2.0
 -->
+<script>
+import { reactive } from 'vue';
+export const vaultsStore = reactive({
+  vaultsOutdated: false,
+});
+
+export const vaultsMutations = {
+  setVaultsOutdated(value) {
+    vaultsStore.vaultsOutdated = value;
+  },
+};
+</script>
+
+<script setup>
+import { computed, onMounted, ref, watchEffect } from 'vue';
+import { CollectionManager, CredentialManager, WalletUser } from '@trustbloc/wallet-sdk';
+import { mapActions, mapGetters, useStore } from 'vuex';
```

Comment:
We don't need to import mapper anymore
```suggestion
import { useStore } from 'vuex';
```